Strategy11/formidable-forms
# Design Enhancements for Field Group Pop-up

This PR introduces design improvements to the field group pop-up, following the Interface Reviews conducted by Razvan.

Figma URLs:
[Link to Figma Design Part One](https://www.figma.com/file/7Kwlv3H2ShYEI0wekWnPBb/Interface-Review-Fixes?type=design&node-id=845-109&t=s47Uuwq8mWWTURWV-4)
[Link to Figma Design Part Two](https://www.figma.com/file/7Kwlv3H2ShYEI0wekWnPBb/Interface-Review-Fixes?type=design&node-id=845-114&t=s47Uuwq8mWWTURWV-4)

## QA URL:
https://qa.formidableforms.com/sherv3/wp-admin/admin.php?page=formidable&frm_action=edit&id=10

## Testing Instructions:
1. Navigate to `WP Admin > Formidable`.
2. Open an existing form or create a new one.
3. Add some fields in the same row.
4. Verify that the field group pop-up option displays as shown in the "After" screenshots below.

## Screenshots:
### Before
<img width="1165" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/69119241/8da7d7da-ebf7-4aae-9c2e-3eab88817b36">
<img width="1160" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/69119241/b3deef10-47ea-49ad-b79f-e1fd3e492500">


### After
<img width="1181" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/69119241/95458598-129f-4304-88cf-8047ef72d597">
<img width="1178" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/69119241/4873d0d9-d29d-4392-9155-6aaabcf74eca">
